public/v2/apps/shopware.json

```diff
@@ -0,0 +1,117 @@
+    "description": "Shopware is a trendsetting ecommerce platform to power your online business. Our ecommerce solution offers the perfect combination of beauty & brains you need to build and customize a fully responsive online store.",
+    "dockerCompose": {
+        "version": "3.3",
+        "services": {
+            "$$cap_appname-db": {
+                "image": "mysql:5.7",
+                "notExposeAsWebApp": "true",
+                "volumes": [
+                    "$$cap_appname-db-data:/var/lib/mysql"
+                ],
+                "restart": "always",
+                "environment": {
+                    "MYSQL_ROOT_PASSWORD": "$$cap_db_pass",
+                    "MYSQL_DATABASE": "shopware",
+                    "MYSQL_USER": "$$cap_db_user",
+                    "MYSQL_PASSWORD": "$$cap_db_pass"
+                }
+            },
+            "$$cap_appname-shopware": {
+                "depends_on": [
+                    "$$cap_appname-db"
+                ],
+                "image": "shyim/shopware:$$cap_sw_version",
+                "restart": "always",
+                "environment": {
+                    "DATABASE_HOST": "srv-captain--$$cap_appname-db:3306",
+                    "DATABASE_URL": "mysql://$$cap_db_user:$$cap_db_pass@srv-captain--$$cap_appname-db:3306/shopware",
+                    "DB_USER": "$$cap_db_user",
+                    "DB_PASSWORD": "$$cap_db_pass",
+                    "APP_URL": "http://$$cap_appname-shopware.$$cap_root_domain",
+                    "APP_SECRET": "$$cap_app_scret",
+                    "INSTANCE_ID": "$$cap_instance_id",
+                    "MAILER_URL": "$$cap_mailer_url",
+                    "INSTALL_LOCALE": "$$cap_sw_locale",
+                    "INSTALL_CURRENCY": "$$cap_sw_currency",
+                    "INSTALL_ADMIN_USERNAME": "$$cap_sw_admin_username",
+                    "INSTALL_ADMIN_PASSWORD": "$$cap_sw_admin_password"
+                },
+                "volumes": [
+                    "$$cap_appname-state:/state",
+                    "$$cap_appname-custom-plugins:/var/www/html/custom/plugins",
+                    "$$cap_appname-files:/var/www/html/files",
+                    "$$cap_appname-logs:/var/www/html/var/logs",
+                    "$$cap_appname-theme:/var/www/html/public/theme",
+                    "$$cap_appname-media:/var/www/html/public/media",
+                    "$$cap_appname-bundles:/var/www/html/public/bundles",
+                    "$$cap_appname-sitemap:/var/www/html/public/sitemap",
+                    "$$cap_appname-thumbnail:/var/www/html/public/thumbnail",
+                    "$$cap_appname-jwt:/var/www/html/config/jwt"
+                ]
+            }
+        }
+    },
+    "instructions": {
+        "start": "Shopware is a trendsetting ecommerce platform to power your online business.",
+        "end": "Shopware is successfully deployed. Please wait a minute before accessing the shop."
```

Comment:
If you are an employee of Shopware and would like to maintain this app, feel free to add your github URl to the instructions.end so that people know how to ask for support.

Reply:
Done :)